core/src/main/java/hudson/model/AbstractProject.java

```diff
@@ -323,7 +323,12 @@ public void onLoad(ItemGroup<? extends Item> parent, String name) throws IOExcep
         builds = buildMixIn.getRunMap();
         triggers().setOwner(this);
         for (Trigger t : triggers()) {
-            t.start(this, Items.currentlyUpdatingByXml());
+            try {
+                t.start(this, Items.currentlyUpdatingByXml());
+            } catch (Exception e) {
+                Logger.getLogger(AbstractProject.class.getName()).log(Level.WARNING,
+                        "could not start trigger '" + t + "' while loading project '" + name + "'", e);
```

Comment:
* Wouldn't `t.getDescriptor().getDisplayName()` be more readable? Or does `t.toString()` add some significant information?
* If possible at this point in time, it should use `getFullName()` instead.